src/bootstrap/src/core/config/config.rs

```diff
@@ -901,7 +900,6 @@ define_config! {
         sign_folder: Option<String> = "sign-folder",
         upload_addr: Option<String> = "upload-addr",
         src_tarball: Option<bool> = "src-tarball",
-        missing_tools: Option<bool> = "missing-tools",
```

Comment:
We should keep this for a while to not break current config files. You can add a FIXME note something like "remove this around mid 2024"